tests/sort_test.py

```diff
@@ -26,3 +26,19 @@ def test_sort_filtered():
     df = vaex.from_arrays(x=x)
     df_sel_sorted = df[df.x > 4].sort(by='x')
     assert df_sel_sorted.x.tolist() == [5, 6, 7]
+
+def test_sort_multikey():
+    x = np.array([5, 3, 1, 1, 5])
+    y = np.array([0, 3, 4, 2, 1])
+    z = np.array(['dog', 'cat', 'cat', 'dog', 'mouse'])
+    df = vaex.from_arrays(x=x, y=y, z=z)
+
+    # Case 1: numeric keys
+    df_sorted_1 = df.sort(by=['x', 'y'])
+    assert df_sorted_1.values.tolist() == [[1, 2], [1, 4], [3, 3], [5, 0], [5, 1]]
```

Comment:
You are assuming that the order of the columns is x, then y. This is not supported before Python3.6. Use from_items to ensure order. Of simply use df_sorted_1.x.tolist() and y separate

Reply:
Ah..i keep on forgetting that. Now there is an assertion for each column.